### PR TITLE
Change setup instructions to use east

### LINKED ISSUE #4: Change "Setup" portion of readme to use east
## Context

Since east now works and is used in our projects, the template should have instructions on how to set up using east.

### PR DESCRIPTION
**Pull request by TjazVracko**
## Description

Since east now works and is used in our projects, the template should have instructions on how to set with east.
I also included some building and flashing instructions. Maybe those are not needed here, but we seem to have them in all of our projects.

Closes #4

## Areas of interest for the reviewer

The readme.

## Checklist

- [x] ~~My code follows the [style guidelines] as defined by IRNAS.~~ No code updated
- [x] I have performed a self-review of my code.
- [x] My changes generate no new warnings.
- [x] I added/updated source code documentation for all newly added or changed functions.
- [x] ~~I updated the CHANGELOG.~~ There is nothing in the changelog. Is that ok?

## After-review steps

* Reviewer can merge and delete the branch.

[style guidelines]: https://github.com/IRNAS/irnas-guidelines-docs/blob/dev/docs/developer_guidelines.md

### CODE REVIEW

**Comment by TjazVracko** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #5** <a href="https://app.graphite.dev/github/pr/IRNAS/irnas-zephyr-template/5" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/IRNAS/irnas-zephyr-template/5?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -22,21 +22,48 @@ of CI workflows for release automation.
...
+east update toolchain
+```
+
+## Building and flashing
+
+To build the application firmware:
+
+```bash
+cd app
+east build -b <board_name> -u <build_type>
```
Comment (MarkoSagadin) [inline README.md:51]: Add a step in the checklist which says to update the values in this command with correct ones.
(what is the board name, is there a build type, etc.)

Now that I am thinking about this, this is something that can really be done before first release.

File: README.md [hunk 2]
```diff
@@ -22,21 +22,48 @@ of CI workflows for release automation.
...
+```
+
+To flash the firmware:
+
+```bash
+east flash --runner jlink
```
Comment (MarkoSagadin) [inline README.md:57]: Runner option is not needed if jink is used by default in the board.cmake file.

File: README.md [hunk 3]
```diff
@@ -22,21 +22,48 @@ of CI workflows for release automation.
...
+```
+
+To view rtt logs:
+
+```bash
+east util connect
+
+# in a new terminal window:
+east util rtt
```
Comment (MarkoSagadin) [inline README.md:66]: Consider changing this to:

```bash
# Run in first terminal window
east util connect
# Run in a second, new terminal window
east util rtt
```

**Review by MarkoSagadin — CHANGES_REQUESTED** [review]
Thank you for adding this! :+1: 

Few minor things below.

**Review by MarkoSagadin — APPROVED** [review]
Nice, squash the last commit and merge

**Pull request merged**